lounge/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+lounge:
+  image: thelounge/lounge:latest
```

Comment:
Just curious, are we talking about pulling the latest version from npm or latest tag from GitHub? In our process, they match, but in reality they aren't related and I'm curious where things come from.